packages/contracts/contracts/voting/majority/IMajorityVoting.sol

```diff
@@ -43,37 +43,93 @@ import "../../core/IDAO.sol";
 ///  For minimal values, $\ge$ comparison is used. This **does** include the minimum participation value. E.g., for $\texttt{minParticipation} = 40\%$ and $N_\text{total} = 10$, the criterion is fulfilled if 4 out of 10 votes were casted.
 /// @dev This contract implements the `IMajorityVoting` interface.
 interface IMajorityVoting {
+    /// @notice Vote options that a voter can chose from.
+    /// @param None The default option state of a voter indicating the absence of from the vote. This option neither influences support nor participation.
+    /// @param Abstain This option does not influence the support but counts towards participation.
+    /// @param Yes This option increases the support and counts towards participation.
+    /// @param No This option decreases the support and counts towards participation.
     enum VoteOption {
         None,
         Abstain,
         Yes,
         No
     }
 
+    /// @notice The vote mode in which the plugin can be configured.
+    /// @param Standard In standard mode, early execution and vote replacement are disabled.
+    /// @param EarlyExecution In early execution mode, a proposal can be executed early if the vote outcome cannot mathematically change by more voters voting.
+    /// @param VoteReplacment In vote replacement mode, voters can change their vote multiple times and only the latest vote option is tallied.
+    enum VoteMode {
+        Standard,
+        EarlyExecution,
+        VoteReplacement
+    }
+
+    /// @notice A container for the plugin-wide proposal vote settings.
+    /// @param voteMode A parameter to select the vote mode.
+    /// @param supportThreshold The support threshold value.
+    /// @param minParticipation The minimum participation value.
+    /// @param minDuration The minimum duration of the proposal vote in seconds.
+    /// @param minProposerVotingPower The minimum voting power required to create a proposal.
+    struct PluginSettings {
```

Comment:
As said before, this name only applies to voting settings, not to plugins in general.

The actual plugin settings could use this shared struct but they all should be individually be using their own custom `____PluginSettings`

Reply:
Fixed in c69aa38770f33a15baed838deca116c0c78f7e22